src/patternfly/components/Card/card.scss

```
@@ -196,6 +196,14 @@
   }
   // stylelint-enable selector-max-class
 
+  // when a card is clickable only, or has the screen reader class on its input
+  &.pf-m-clickable:not(.pf-m-selectable),
+  &:has(input.#{$pf-prefix}u-screen-reader) {
```

You'd probably want a more narrow scope on the `input` to be able to handle nested cards. `> .#{$card}__selectable-actions` is optional but I'd include it because who knows if someone has some other thing in their card actions that could trigger this style change.

```suggestion
  &.pf-m-clickable:not(.pf-m-selectable),
  &:has(> .#{$card}__header > .#{$card}__actions > .#{$card}__selectable-actions input.#{$pf-prefix}u-screen-reader) {
```

It's probably more efficient to write it this way though (not tested) since targeting the input only needs to start at the `__actions` container?

```suggestion
  @at-root .#{$card}__actions:has(> .#{$card}__selectable-actions input.#{$pf-prefix}u-screen-reader),
  &.pf-m-clickable:not(.pf-m-selectable) {
```